### PR TITLE
dont embed v1.Error as it contains a mutex; copy/paste its struct values

### PR DESCRIPTION
**Pull request by jacobweinstock**
Signed-off-by: Jacob Weinstock <jakobweinstock@gmail.com>

## Description

don't embedding v1.Error as it contains a mutex. 

## Why is this needed

fix failed builds (go vet; golangci-lint)

Fixes: #

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [ ] added unit or e2e tests
- [ ] provided instructions on how to upgrade

### CODE REVIEW

**Review by mikemrm — APPROVED** [review]
These changes once merged will cause #21 to have conflicts as at least some of this was also done in that pr

File: pkg/oob/oob.go [hunk 1]
```diff
@@ -25,5 +23,11 @@ type BMC interface {
 
 // Error for all bmc actions
 type Error struct {
```
Comment (mmlb) [inline pkg/oob/oob.go:25]: hmm would an `alias` work here instead?

Comment (jacobweinstock) [reply pkg/oob/oob.go:25]: `go vet` will still complain.

**Comment by jacobweinstock** [comment]
> These changes once merged will cause #21 to have conflicts as at least some of this was also done in that pr

ah you're right. i'll close this and just wait for https://github.com/tinkerbell/pbnj/pull/21 to merge

**Pull request closed without merging**